R/wb_functions.R

```diff
@@ -521,13 +521,13 @@ wb_to_df <- function(
   # text in t
   if (any(cc_tab %in% c("inlineStr"))) {
     sel <- cc$c_t %in% c("inlineStr")
-    cc$val[sel] <- is_to_txt(cc$is[sel])
+    cc$val[sel] <- replaceXMLEntities(is_to_txt(cc$is[sel]))
```

Comment:
I'm not entirely happy with this. It should work reasonably well, but maybe we can check it we have to replace something in the first place.